testdata/p4_16_samples_outputs/issue363-bmv2.p4

```diff
@@ -0,0 +1,68 @@
+#include <core.p4>
```

Comment:
this code does not test any of the new language features.

Reply:
Will add more test cases with next commit

Reply:
In regards to testing any new language features, this is what has been done today.  Please let us know what other language tests would you like. 

1. issue383-bmv2.p4 (note issue363-bmv2.p4 has been renamed) has been beefed up for following tests:

(a) struct that has both nested structs and bitvec headers.
(b) bit-vector headers that lie deeper within nested struct. struct->struct->bit-vector header. 

For the new tests, both local variables and parameters to control/parser are tested.  The references are also tested.  

Then, I copied issue383-bmv2.p4 to bitvec_hdr.p4 and tested bitvec_hdr.p4 with p4test.